test/dynamo/test_export.py

```diff
@@ -2842,8 +2842,8 @@ def f(x):
 
         example_inputs = (torch.rand(5),)
         with self.assertRaisesRegex(
-            torch._dynamo.exc.UserError,
-            "Expected 4 arguments",
+            torch._dynamo.exc.ArgsMismatchError,
```

Comment:
Was this expected to be UserError in export DB?

Reply:
Not yet.